pkg/operator/controller/testdata/tls_with_non_existing_cabundle

```diff
@@ -0,0 +1,25 @@
+.:5353 {
+    bufsize 512
+    errors
+    log . {
+        class error
+    }
+    health {
+        lameduck 20s
+    }
+    ready
+    kubernetes cluster.local in-addr.arpa ip6.arpa {
+        pods insecure
+        fallthrough in-addr.arpa ip6.arpa
+    }
+    prometheus 127.0.0.1:9153
+    forward . tls://9.8.7.6 tls://[1001:AAAA:BBBB:CCCC::2222]:53 {
+        tls_servername example.com
+        tls
```

Comment:
I wonder if this entire line should be removed.  What happens when just `tls` is in the config?

Reply:
When only using `tls`, CoreDNS will use system certs to validate the server certificate of the upstream resolver. The docs are not terribly verbose on this but in manual and e2e testing, having `tls` alone on a line works: https://coredns.io/plugins/forward/